examples/Podfile.lock

```diff
@@ -11,8 +11,8 @@ EXTERNAL SOURCES:
     :path: "../"
 
 SPEC CHECKSUMS:
-  MDFTesting: 7cc0f91ee418509f4290d0fb3a491b0bb5d1efbc
+  MDFTesting: d24689ef4322740a4a438c53353c9f5374591514
 
 PODFILE CHECKSUM: f68cc4584957218d15c617848be26ebbd4bf5c8f
 
-COCOAPODS: 1.7.2
+COCOAPODS: 1.8.4
```

Comment:
Looks like this might have been an unintentional change. Please revert or include it in the PR summary.

Reply:
We should be cautious about mixing Cocoapods versions. MDC-iOS only supports 1.6.1.

Reply:
Done.